app/client/src/sagas/JSPaneSagas.ts

```
@@ -539,32 +537,9 @@ export function* handleExecuteJSFunctionSaga(data: {
       );
     }
 
-    if (!!collection.isMainJSCollection)
+    if (!!collection.isMainJSCollection) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)